htable: feature parity with kemi funcs

devopsec opened the pull request:
- add sht_is_null route function
  - add sht_inc route function
  - add sht_dec route function

#### Pre-Submission Checklist
- [x] Commit message has the format required by CONTRIBUTING guide
- [x] Commits are split per component (core, individual modules, libs, utils, ...)
- [x] Each component has a single commit (if not, squash them into one commit)
- [x] No commits to README files for modules (changes must be done to docbook files
in `doc/` subfolder, the README file is autogenerated)

#### Type Of Change
- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)

#### Checklist:
- [ ] PR should be backported to stable branches
- [x] Tested changes locally
- [ ] Related to issue #XXXX (replace XXXX with an open issue number)

#### Description
Implementing the identical functions from KEMI in the route functions.

miconda commented:
The Kemi functions were added to correspond to operations with variables.

```
$shv(x=>y) == $null
$shvinc(x=>y) 
$shvdec(x=>y) 
```

I am not really against adding also functions, but returning 0 to kamailio.cfg, as I could see quickly in the docs from commit, stops the execution of config.

devopsec commented:
@miconda I sepcified in the docs the return was "< 0" but I understand that was not as clear at a glance.  
I updated that portion of the doc to be more explicit and l also moved the return value for parameter parsing errors to 3 instead (reviewing this I saw it conflicted with the meaning of -1 return value).
